src/ripple/app/paths/cursor/RippleLiquidity.cpp

```diff
@@ -129,22 +129,18 @@ void rippleLiquidity (
         // If the quality is worse than the previous
         JLOG (rippleCalc.j_.trace()) << "rippleLiquidity: Fee";
 
-        std::uint64_t uRate = getRate (
-            STAmount (uQualityOut), STAmount (uQualityIn));
+        std::uint64_t const uRate = getRate (
```

Comment:
Fabulous cleanup so far.  Bummer that the parameter `uRateMax` is passed as a `uint64_t`.  If `uRateMax` were passed as a `Rate`, then `uRate` here could be a `Rate`.  But it can wait.